salt/utils/asynchronous.py

```diff
@@ -105,7 +103,26 @@ def close(self):
                 log.exception("Exception encountered while running stop method")
         io_loop = self.io_loop
         io_loop.stop()
-        io_loop.close(all_fds=True)
+        try:
+            io_loop.close(all_fds=True)
+        except FileNotFoundError:
+            # Traceback (most recent call last):
```

Comment:
Should throw some sort of logging in here. I'm not sure exactly what, but... something :thinking:

Reply:
Doesn't work :)